Core/Utilities/ClassAd/ClassAdLight.py

```diff
@@ -254,7 +254,7 @@ def asJDL(self):
     """
 
     result = ''
-    for name, value in self.contents.items():
+    for name, value in sorted(self.contents.items()):
```

Comment:
Why `sorted`?

Reply:
As the tests for the JDL and XML compare the raw bytes in the files. As there is no ordering of dictionaries in Python 2.7 this is a quirk of the interpreter state at the particular time the file is written.

In ef16451eaaf10eac817941ddbd30a64b6ae21d7a I make the XML and JDL deterministic and update the references accordingly.